### DISCUSSION TITLE
Responsive image conversion queued even if QUEUE_CONVERSIONS_BY_DEFAULT=false

### DISCUSSION THREAD

**Discussion by VisionovaDev** [spatie/laravel-medialibrary#3659] [Q&A]
Hi,
I set QUEUE_CONVERSIONS_BY_DEFAULT=false in the .env file but when I add an image using  this code:
```
  $category->addMedia($imgFileToImport)                               
                                ->preservingOriginal()
                                ->withResponsiveImages()
                                ->toMediaCollection('main_image');
```
a GenerateResponsiveImagesJob is queued ignoring the settings of the .env  file. 
I cleared all the caches
It seems that the responsive image conversions are ALWAYS queued even if QUEUE_CONVERSIONS_BY_DEFAULT is set to false

**Comment by SquareNetMedia** [comment]
I have the same issue
